### DISCUSSION TITLE
Error after library update

### DISCUSSION THREAD

**Discussion by SandeepJagdev** [pschatzmann/arduino-audio-tools#489] [Q&A] [answered]
Hello i upadated library with recent update. previous code Start show error after update. Even A2DP examples also show this error.
         void flush() override  {

              ^
 warning: narrowing conversion of 'AudioKitConfig::sampleRate()' from 'uint32_t {aka unsigned int}' to 'int' inside { } [-Wnarrowing]

         .sample_rate = sampleRate(),

                                  ^

**Comment by pschatzmann** [comment, marked as answer]
I don't see any error: as you can read above this is just a warning!
If you check in the board manager, what ESP32 version are you using ?
Since I do not get this warning I assume you use a very old, obsolete version. The latest is 2.0.5!
